### ISSUE TITLE
No one ports from PC

### ISSUE THREAD

**Issue by jv110** [MrCrayfish/MrCrayfishFurnitureMod-MCPE#38]
I'm noticing most renders people do are either just wrong or really awful, and they don't even look like they are in PC version. An example is the Oven, it's weird, has wrong textures and dimensions and there's even a missing obsidian. This mod is supposed to be a port of MrCrayfish's Furniture Mod on PC for MCPE, not a different mod, so my suggestion is you guys just port from PC!

**Comment by MrCrayfish** [comment]
Yes, I up-vote this request. Anyone who does port furniture over, look at the model code in the PC version instead of guessing. I will fix up any funky renders
